Documentation/BuildInstructionsLadybird.md

```diff
@@ -176,7 +176,7 @@ pkgman install cmake cmd:python3 ninja openal_devel qt6_base_devel qt6_multimedi
 
 ### Android:
 
-On a Unix-like platform, install the prerequisites for that platform and then see the [Android Studio guide](AndroidStudioConfiguration.md).
```

Comment:
A few of these files have embedded or referenced resources. images, xml files, etc.

Reply:
Do you have examples of ones that are wrong? I just gave it a look through and it seems like CLion correctly updated all the links.

Reply:
Nope, it actually looks correct! I thought we had more image files, but I guess I removed those a while back.